tests/shared/src/test/scala/scala/meta/tests/prettyprinters/SyntacticSuite.scala

```diff
@@ -660,4 +660,28 @@ class SyntacticSuite extends scala.meta.tests.parsers.ParseSuite {
     assert(q"list map (add(_, 1))".syntax == "list map (add(_, 1))")
     assert(q"list map (bar:_*)".syntax == "list map (bar: _*)")
   }
+  test("#1826 ApplyInfix parentheses on Select") {
+    assert(q"list map (_.bar)".syntax == "list map (_.bar)")
```

Comment:
Can we introduce a helper method to run those assertions and verify the AST is unchanged?
```
def check(original: Tree, expected: String): Unit = {
  assertNoDiff(original.syntax == expected)
  assertNoDiff(original.structure = parseStat(expected).structure)
}
```

Reply:
Done